addons/website_mass_mailing/static/src/scss/website_mass_mailing.scss

```diff
@@ -0,0 +1,7 @@
+.s_newsletter_subscribe_form {
+    @include media-breakpoint-up(md) {
+        .o_subscribe_input {
+            max-width: 20rem;
+        }
+    }
+}
```

Comment:
Technically should create an ir.asset, a versioning number etc... but I guess we can treat it as a "generic" option as you did. I'll at least rename the class to state its purpose then weird to have 4 newsletter snippets but only one with a class named "o_subscribe_input"...